database/MysqlDB.py

```diff
@@ -1,37 +1,38 @@
-import time
-import json
-from global_logger import logger, VERBOSE
 import inspect
-#This class holds all the mysql syntax for the sql class
-## TODO: change MysqlDB to db and change querry builder to mysqlQuerry
-class MysqlDB:
+import time
+
+from global_logger import logger
```

Comment:
Needs to import VERBOSE as well. Python logging doesn't have a built in VERBOSE method so I created one with a log level of 15. Logger goes from info, to warning to error... wanted something that wasn't a warning but not quite info level. 
https://docs.python.org/3/library/logging.html#logging-levels

Reply:
I'll fix that now

Reply:
I'll fix it... I don't get why this was missing... my local branch has the VERBOSE import here